gammapy/modeling/models/spatial.py

```diff
@@ -159,11 +159,6 @@ def integrate_geom(self, geom):
 
         return Map.from_geom(geom=geom, data=data.value, unit=data.unit)
 
-    @property
-    def evaluation_radius(self):
```

Comment:
Why did you remove this? The purpose is to provide a default for custom implemented model, so that users are not required to implement an evaluation radius. But it was obviously missing a unit test...

Reply:
It was duplicated

Reply:
Thanks, indeed! I missed that...